app.rb

```diff
@@ -36,14 +42,16 @@ def create_person
     case selected_person
     when 1
       print 'Has parent permission? [Y/N]: '
+
       provided_permission = gets.chomp.capitalize
       student_permission = true if provided_permission == 'Y'
       student_permission = false if provided_permission == 'N'
-      @people.push(Student.new(age, name, parent_permission: student_permission))
+      @people.push(Student.new(age, student_permission, name))
+
     when 2
       print 'Specialization: '
       specialization = gets.chomp.to_s
-      @people.push(Teacher.new(age, name, specialization))
+      @people.push(Teacher.new(specialization, age, name))
```

Comment:
- [ ] When I register one teacher after I leave the app the teacher's name is not a person on the JSON file. The other objects like specialization and age are but name not. Kindly check into that

Reply:
- [ ] People's ID's are not showing here, kindly update it 
![image](https://user-images.githubusercontent.com/62705947/190441801-0c80d0b1-0bd1-4159-81bb-686e15928828.png)